crates/router/src/core/payment_methods/cards.rs

```diff
@@ -208,6 +246,60 @@ pub async fn update_customer_payment_method(
 
 // Wrapper function to switch lockers
 
+pub async fn add_bank_to_locker(
+    state: &routes::AppState,
+    req: api::PaymentMethodCreate,
+    merchant_account: &domain::MerchantAccount,
+    key_store: &domain::MerchantKeyStore,
+    bank: &api::BankPayout,
+    customer_id: &String,
+) -> errors::CustomResult<(api::PaymentMethodResponse, bool), errors::VaultError> {
+    let key = key_store.key.get_inner().peek();
+    let payout_method_data = api::PayoutMethodData::Bank(bank.clone());
+    let enc_data = async {
+        serde_json::to_value(payout_method_data.to_owned())
+            .into_report()
+            .change_context(errors::VaultError::SavePaymentMethodFailed)
+            .attach_printable("Unable to encode payout method data")
+            .ok()
```

Comment:
Are we sure we want to ignore the error here? Please ensure error propagation is not a requirement for this flow. If it's not a requirement, please log the error `crate::logger::error!()`.

Reply:
I have updated the code, and now logging the error before changing context.